public/index.html

```diff
@@ -7,9 +7,9 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="Pablo Felipe's Personal Portfolio - Made with Tailwind"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="p-icon" href="%PUBLIC_URL%/icon.png" />
```

Comment:
I think is better use one of normal rel like the [documentation](https://www.w3schools.com/tags/att_link_rel.asp)

Reply:
Oh, very cool. Changed!